src/components/input-group/input-group.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import inputGroupDemo from './demo/input-group-demo.twig';
+import './input-group.scss';
+import './demo/styles.scss';
+
+<Meta title="Components/Input Group" />
```

Comment:
I think this might actually be an Object?

What do you think, @Paul-Hebert @gerardo-rodriguez ?

Reply:
Hmm, good question. I think I'd lean slightly towards this being an object since its focus is on styling/laying out arbitrary child elements. This feels similar to the Button Group which is an object.

Reply:
Yeah, I agree with @Paul-Hebert's assessment.